content/webapp/pages/search/index.tsx

```diff
@@ -400,7 +455,9 @@ export const getServerSideProps: GetServerSideProps<
         | WellcomeApiError
         | undefined,
       stories,
-      events;
+      events,
+      works,
+      images;
```

Comment:
I think we could/should keep those typed, which is lost and would currently be `any`. I know I did that for stories and events and honestly I can't quite remember why so I'm at fault, really 😅